### DISCUSSION TITLE
[FATAL] Unable to load the configuration file [dnscrypt-proxy.toml]

### DISCUSSION THREAD

**Discussion by Tetonne** [DNSCrypt/dnscrypt-proxy#2089] [Q&A] [answered]
error : [FATAL] Unable to load the configuration file [dnscrypt-proxy.toml] -- Maybe use the -config command-line switch?

brew update; brew upgrade give : 

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Library/Frameworks/Python.framework/Versions/2.7/bin/python2-config
  /Library/Frameworks/Python.framework/Versions/2.7/bin/python2.7-config
  /Library/Frameworks/Python.framework/Versions/2.7/bin/python-config

## Output of the following commands:

config : /usr/local/etc/dnscrypt-proxy.toml
dnscrypt-proxy -version
2.1.1

dnscrypt-proxy -check
[FATAL] Unable to load the configuration file [dnscrypt-proxy.toml] -- Maybe use the -config command-line switch?

dnscrypt-proxy -resolve apple.com
[FATAL] Unable to load the configuration file [dnscrypt-proxy.toml] -- Maybe use the -config command-line switch?

**Comment by jedisct1** [comment, marked as answer]
> `[FATAL] Unable to load the configuration file [dnscrypt-proxy.toml] -- Maybe use the -config command-line switch?`

No `dnscrypt-proxy.toml` file exists in the directory you are currently in.

Either go to the directory that contains such file, or use the `-config` command-line switch to provide the path to the configuration file:

```sh
dnscrypt-proxy -config /usr/local/etc/dnscrypt-proxy.toml
```

Homebrew also comes with an `info` command that often gives useful information about how to use a package:

```sh
brew info dnscrypt-proxy
```

**Reply by Tetonne** [reply]
thanks for your quick answer and solution. best regards.
